ui/admin-portal/src/app/components/account/verify-email/verify-email.component.html

```diff
@@ -0,0 +1,100 @@
+<div class="modal fade" id="verifyEmailModal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered">
```

Comment:
This is a very non standard way of doing modals in Angular.

Our standard approach is to use NgbModal. See lots of examples in our code - plus here https://ng-bootstrap.github.io/#/components/modal/examples

Reply:
I didn't realize the approach was so non-standard! Apologies, I’ll update it!